src/snowcli/app/cli_app.py

```diff
@@ -35,6 +37,15 @@
 _commands_registration = CommandsRegistrationWithCallbacks(_api.plugin_config_provider)
 
 
+@dataclass
+class AppContextHolder:
+    # needed to access the context from tests
+    app_context: Optional[Context] = None
```

Comment:
I'm not sure if we should add something to code just for testing

Reply:
That's a good point, but I don't see any other alternative.

Reply:
The alternative is to add callback option. So maybe we should wait for @sfc-gh-turbaszek decision - which is better?

Reply:
Yes, it does - we get the list of params and commands, so it`s easy to check, if help is present

Reply:
Ok but, am I wrong thinking that the callback is also only for purpose of these tests?

Reply:
Yes it is

Reply:
During our today's meeting we agreed that we accept my solution.